cornell-dti/course-plan
# Public policy minor

### Summary 

This pull request adds the public policy minor to CoursePlan

![image](https://user-images.githubusercontent.com/25535093/113499488-4d950c00-94e4-11eb-9b41-4f23196a7ba1.png)

### Test Plan 

1. Confirm that the minor can be completely fulfilled.
2. Confirm that the gateway course and methods courses cannot be used as electives

### Notes 

Just a couple things

- When trying to exclude 'GOVT 3032' from the electives checker, it still could fulfill it until I also excluded its cross-listed course code. Is this intentional / can we fix this?
- This is the first requirement I've written from scratch, so please let me know if I did anything incorrectly!
